EndlessLauncher/ViewModel/MainViewModel.cs

```diff
@@ -164,46 +125,7 @@ public RelayCommand OpenKolibriRelayCommand
             {
                 if (openKolibriRelayCommand == null)
                 {
-                    openKolibriRelayCommand = new RelayCommand(() =>
-                    {
-                        // Show Kolibri window if already exists.
-                        // Ideally, this should be done in the Kolibri app but for now we
-                        // stick with this approach.
-                        if (Utils.ActivateWindow("wxWindowNR", "Kolibri"))
-                        {
-                            return;
-                        }
```

Comment:
Is this already implemented by Kolibri then? Could this help with the race conditions that Matt was probably experiencing?

Reply:
Indeed, this is implemented by Kolibri now: https://github.com/learningequality/kolibri-installer-mac/commit/82af6c08ae5bb8788c1769d5aec7f95b3be84237. It appears to be working as expected, although I just realized it doesn't raise the existing window, which is a bit unfortunate, so maybe I should leave that in for now. It's off topic for this branch anyway :)